apps/src/makerlab/dropletConfig.js

```diff
@@ -18,30 +18,35 @@ const playSongConfig = {
   tempo: 100000
 };
 
-const pixelType = '[Pixel]';
-const colorPixelVariables = ['pixels', 'pixel0', 'pixel1', 'pixel2', 'pixel3', 'pixel4', 'pixel5', 'pixel6', 'pixel7', 'pixel8', 'pixel9'];
-const simpleLedVariables = ['led', ...colorPixelVariables];
+const pixelType = '[ColorLed].';
+const colorPixelVariables = Array.from({length: 10}, (_, index) => `colorLeds[${index}]`);
```

Comment:
could also be done as `_.range(10).map(index => colorLeds[${index}]` which I personally find marginally easier to read than Array.from({length: 10})